app/org/sagebionetworks/bridge/models/surveys/BloodPressureConstraints.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.sagebionetworks.bridge.models.surveys.UIHint.BLOODPRESSURE;
+
+import java.util.EnumSet;
+
+public class BloodPressureConstraints extends Constraints {
+    public BloodPressureConstraints() {
+        setDataType(DataType.STRING);
```

Comment:
It turns out DataType is coupled to how Constraints are deserialized, so I added values for BLOODPRESSURE, WEIGHT, and HEIGHT